src/containers/BooksForm.js

```diff
@@ -1,24 +1,87 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createBook } from '../actions/index';
 
-export const BooksForm = () => {
-  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
-
-  return (
-    <form>
-      <label htmlFor="title-text">
-        Title:
-        <input id="title-text" type="text" />
-      </label>
-
-      <label htmlFor="book-categories">
-        Categories:
-        <select name="categories" id="book-categories">
-          {categories.map(x => <option key={x} value={x}>{x}</option>)}
-        </select>
-      </label>
-      <button type="submit">Submit</button>
-    </form>
-  );
+import { getRandomInt } from '../utils/getRandomInt';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: categories[0],
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) {
+    e.preventDefault();
+    if (e.target.name === 'title') {
+      this.setState({
+        title: e.target.value,
+      });
+    } else {
+      this.setState({
+        category: e.target.value,
+      });
+    }
```

Comment:
You could combine this into one. 
```suggestion
    this.setState({
       [e.target.name]: e.target.value,
    });
```